lms/djangoapps/class_dashboard/tests/test_dashboard_data.py

```diff
@@ -85,6 +98,36 @@ def setUp(self):
                     state=json.dumps({'attempts': self.attempts}),
                 )
 
+            # adding an instructor and a staff to the site. They should not be counted. Test should not fail
+            StudentModuleFactory.create(
+                grade=1,
+                max_grade=1,
+                student=instructor_member,
+                course_id=self.course.id,
+                module_state_key=self.item.location,
+                state=json.dumps({'attempts': self.attempts}),
+            )
+
+            StudentModuleFactory.create(
+                grade=1,
+                max_grade=1,
+                student=staff_member,
+                course_id=self.course.id,
+                module_state_key=self.item.location,
+                state=json.dumps({'attempts': self.attempts}),
+            )
+
+            # Adding this student will fail the test
+
+            # StudentModuleFactory.create(
+            #     grade=1,
+            #     max_grade=1,
+            #     student=newstudent,
+            #     course_id=self.course.id,
+            #     module_state_key=self.item.location,
+            #     state=json.dumps({'attempts': self.attempts}),
+            # )
+
```

Comment:
It was actually useful to see it during code review, for me at least.  But we try to commit code with as little comment-out as possible.